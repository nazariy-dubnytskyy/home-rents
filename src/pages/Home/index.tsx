import * as React from 'react';
import styled from 'styled-components';

export const Wrapper = styled.div`
  padding: 0 ${props => props.theme.spaceUnit.two};
`;

function Home() {
  return (
    <Wrapper>
      This is implementation of <a href="https://www.notion.so/Frontend-Challenge-vo2-2020-bea9e123187f405e9c58db8c9e5f0320">Frontend Challenge vo2 - 2020.</a>
    </Wrapper>
  )
}

export default Home;