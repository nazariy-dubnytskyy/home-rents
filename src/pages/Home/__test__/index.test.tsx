import * as React from 'react';
import '@testing-library/jest-dom';
import '@testing-library/jest-dom/extend-expect';

import { render } from '../../../utils/test-utils';
import Home from '..';

describe('Home page', () => {
  it('renders correctly', () => {
    const { container } = render(<Home />);

    expect(container.firstChild).toMatchSnapshot();
  });
});