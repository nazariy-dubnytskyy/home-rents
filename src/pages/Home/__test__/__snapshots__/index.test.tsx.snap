// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home page renders correctly 1`] = `
<div
  class="sc-AxjAm gDzslF"
>
  This is implementation of 
  <a
    href="https://www.notion.so/Frontend-Challenge-vo2-2020-bea9e123187f405e9c58db8c9e5f0320"
  >
    Frontend Challenge vo2 - 2020.
  </a>
</div>
`;
